docs/dasharo-menu-docs/dasharo-system-features.md

```diff
@@ -19,6 +19,19 @@ This menu offers security-sensitive options like:
   BIOS flash. Recommended to be enabled if [Firmware setup password](overview.md#user-password-management)
   is set. Disable the protection if you need access to flash with flashrom.
 
+- `Dma protection` - Early Boot DMA protection is a feature that utilizes the
+  IOMMU early in the boot process to prevent rogue DMA-capable devices such as
+  PCIe add-in cards and USB 3 devices from access to memory. This prevents a
+  class of DMA attacks that allow for exfiltration of secrets and installation
+  of malware early in the boot process. DMA protection can be enabled in two
+  ways:
+- `IOMMU handoff at ExitBootServices` - this option is available when
+  `Dma protection` is enabled. It changes the moment when IOMMU is re-enabled.
+  If this option is selected, the it's re-enabled at ExitBootService when
```

Comment:
```suggestion
  If this option is selected, then it's re-enabled at ExitBootService when
```

Reply:
`ExitBootService` or `ExitBootService` ? The description in the first line of this feature is different

Reply:
It's `ExitBootServices`, fixed the other accordingly.